vcpkg.json

```diff
@@ -25,6 +25,47 @@
         "zstd"
       ]
     },
+    {
+      "name": "dawn",
+      "platform": "android",
+      "default-features": false,
+      "features": [
+        "vulkan"
+      ]
+    },
+    {
+      "name": "dawn",
+      "platform": "freebsd | openbsd",
```

Comment:
Wayland is also support on FreeBSD and OpenBSD, so maybe this should just get merged with the Linux case, if not this is still a duplicate with Android.